indico/modules/events/models/events.py

```diff
@@ -1110,6 +1110,14 @@ def is_user_speaker(self, user):
                 ))
                 .has_rows())
 
+    def _get_email_templates_rules(self):
```

Comment:
I don't think this should be in the Event model, maybe put it as a normal function in indico/modules/events/abstracts/util.py